.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1 +1,11 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Create an issue
```

Comment:
This still can be not notable enough.
Maybe name = "Report a bug or request a feature", about = "Use YouTrack for new issues" ?